client/python/armada_client/client.py

```diff
@@ -49,7 +49,7 @@ def get_job_events_stream(
         queue: str,
         job_set_id: str,
         from_message_id: Optional[str] = None,
```

Comment:
Could you just assign this to ""?  So you can remove the check below?

Reply:
You could just default initialize it to “” rather than optional?  Not sure if that’s normal in python

Reply:
Its because the function that we pass that to expects that type, so my hands are tied I think.